diff --git a/gtest/CMakeLists.txt.in b/gtest/CMakeLists.txt.in
index d3c1d55..a1f1c45 100644
--- a/gtest/CMakeLists.txt.in
+++ b/gtest/CMakeLists.txt.in
@@ -5,11 +5,11 @@ project(googletest-download NONE)
 include(ExternalProject)
 ExternalProject_Add(googletest
         GIT_REPOSITORY    https://github.com/google/googletest.git
-        GIT_TAG           master
+        GIT_TAG           main
         SOURCE_DIR        "${CMAKE_CURRENT_BINARY_DIR}/googletest-src"
         BINARY_DIR        "${CMAKE_CURRENT_BINARY_DIR}/googletest-build"
         CONFIGURE_COMMAND ""
         BUILD_COMMAND     ""
         INSTALL_COMMAND   ""
         TEST_COMMAND      ""
-        )
\ No newline at end of file
+        )
